### ISSUE TITLE
display water throughput (l/min)

### ISSUE THREAD

**Issue by ralfhauser** [mmmatthew/water-fountains.org#158]
as suggested as 1) in https://github.com/water-fountains/proximap/issues/152

Icon could be 
![image](https://user-images.githubusercontent.com/1681677/55309937-7864a980-545f-11e9-8138-213f65fbc5b7.png)
or 
![image](https://user-images.githubusercontent.com/1681677/55309954-887c8900-545f-11e9-991f-5a4d8495e698.png)

**Comment by mmmatthew** [comment]
Just to be clear about what this issue is about: you are suggesting to display the water flow value in the detailed view with an icon, and not only in the expandable menu.

Is that accurate? If so, I would modify specify this in the issue title.

**Comment by mmmatthew** [comment]
Also: how literally do you want me to take your icon style and idea suggestions?

I would think a simple gauge is more fitting: 
![image](https://user-images.githubusercontent.com/10126798/55406623-bd223a80-555c-11e9-981f-16b21d9c7313.png)

**Comment by ralfhauser** [comment]
yes, please display icon and number litres per minute as in 
https://www.openstreetmap.org/node/1638729587 on a line in the detail view.

Your icon is fine too

**Comment by ralfhauser** [comment]
Yes - in https://beta.water-fountains.org/zurich?lang=de&queryType=byId&database=wikidata&idval=Q55166045 e.g. in any of the below 4 places:
![image](https://user-images.githubusercontent.com/1681677/55434295-9503fd00-5597-11e9-8ac6-ef69fb00b2a5.png)

**Comment by ralfhauser** [comment]
see also https://taginfo.openstreetmap.org/keys/wetap%3Aflow#overview

**Comment by ralfhauser** [comment]
https://opensensemap.org/about seems a place where IoT measurments could be placed

Basel seems to opload river data to github multiple times per day (Adrian Gschwend, Zazuko Biel)
